src/main/java/org/openmrs/addonindex/backend/Bintray.java

```diff
@@ -65,7 +69,36 @@ public AddOnInfoAndVersions getInfoAndVersionsFor(AddOnToIndex addOnToIndex) thr
 			return handlePackageJson(addOnToIndex, json);
 		}
 	}
-	
+
+	@Override
+	public void fetchDownloadCounts(AddOnToIndex toIndex, AddOnInfoAndVersions infoAndVersions) throws Exception {
+		//We won't be checking for existing info as download counts is dynamic
+		// and has to be fetched each time
+		logger.info("Fetching Download Counts for " + toIndex.getUid());
+		BintrayDownloadCounts downloadCounts;
+		String json;
+		BintrayPackageDetails bintrayPackageDetails = toIndex.getBintrayPackageDetails();
+		String baseurl = "https://bintray.com/statistics/packageGeoStats?&pkgPath=/" + bintrayPackageDetails.getOwner()
```

Comment:
Why is this "baseurl"? Base implies that you're going to append something to it. Call it downloadStatsUrl or something.